content/language/word-list.md

```diff
@@ -2,6 +2,61 @@
 title: "Word replacement list"
 ---
 
+## Abort, abortion
+
+**Recommended Replacement Terms**
+
+Depending on context, the INI recommends the following alternative terms.
+
+- User-Initiated Termination
```

Comment:
None of these quite capture the meaning of the C `abort` function, which is terminate ASAP and try to leave debugging information (a core file) behind. There is also the Unix/Linux `SIGABRT` signal.

Reply:
@richsalz Is the issue at play this particular replacement ("User initiated termination") or the overall idea of replacing abort? 

If the latter, I think we cover this in https://github.com/inclusivenaming/website/pull/93/files#diff-598289ba4c816c1911e71d1984cd08af134c6205fe27d502f7cf090fb95fcd85R41

Reply:
I'm fine with the commit you pointed to.  Thanks for listening. 

I wonder if there's interest in reviewing common signals, syscalls, library functions, etc., for Linux and suggesting common alternative "aliases" for problematic names?

Reply:
@richsalz  I think there could be, but perhaps a version 2?